airbyte-webapp/src/packages/cloud/views/auth/SignupPage/components/SignupForm.tsx

```diff
@@ -194,9 +188,31 @@ export const SignupFormStatusMessage: React.FC = ({ children }) => (
   <div className={styles.statusMessage}>{children}</div>
 );
 
+const useExperimentallyHiddenField = (experimentKey: keyof Experiments) => {
+  const hideField = useExperiment(experimentKey, false);
+  let fieldSchema = yup.string();
+  if (!hideField) {
+    fieldSchema = fieldSchema.required("form.empty.error");
+  }
+  return { hideField, fieldSchema };
+};
+
 export const SignupForm: React.FC = () => {
   const { signUp } = useAuthService();
 
+  const { hideField: hideName, fieldSchema: nameSchema } = useExperimentallyHiddenField("authPage.signup.hideName");
+  const { hideField: hideCompanyName, fieldSchema: companyNameSchema } = useExperimentallyHiddenField(
+    "authPage.signup.hideCompanyName"
+  );
+
+  const SignupPageValidationSchema = yup.object().shape({
```

Comment:
var name should be camel case, not sure why it was pascal case in the first place.